src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Random random = new Random();
+
+    public static Color getRandomColor() {
```

Comment:
[Don't use static methods in your solution](https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#dont-use-static-methods-in-your-solution)
```suggestion
    public String getRandomColor() {
```

Reply:
not fixed - usage of static is discouraged (it will become clear why when you start Spring Boot part)